src/components/Main/Map.module.css

```diff
@@ -0,0 +1,3 @@
+.mapContainer {
+  height: 96rem;
+}
```

Comment:
Please try to avoid these hard-coded large values. I think this means the whole screen, so suggest using 100vh etc.